src/main/java/com/hackclub/hccore/advancements/MileAdv.java

```diff
@@ -8,16 +8,27 @@
 import com.fren_gor.ultimateAdvancementAPI.util.CoordAdapter;
 import com.hackclub.hccore.HCCorePlugin;
 import org.bukkit.Material;
+import org.bukkit.Statistic;
+import org.bukkit.event.player.PlayerStatisticIncrementEvent;
 
-public class MillionAdv extends BaseAdvancement {
-    static AdvancementDisplayBuilder<AdvancementDisplay.Builder, AdvancementDisplay> displayBuilder = new AdvancementDisplay.Builder(Material.ELYTRA, "Million Miler")
+public class MileAdv extends BaseAdvancement {
+    static AdvancementDisplayBuilder<AdvancementDisplay.Builder, AdvancementDisplay> displayBuilder =
+            new AdvancementDisplay.Builder(Material.ELYTRA, "I'm Gonna Be")
             .challengeFrame()
             .announceChat()
             .showToast()
-            .description("Fly one million miles (1,609,344 km) with an elytra");
+            .description("Just to be the man who walked a thousand miles (1609.344 km) to fall down at your door");
     static int maxProgression = 1;
 
-    public MillionAdv(HCCorePlugin plugin, Advancement root, AdvancementKey key, CoordAdapter adapter) {
+    public MileAdv(HCCorePlugin plugin, Advancement root, AdvancementKey key, CoordAdapter adapter) {
         super(key.getKey(), displayBuilder.coords(adapter, key).build(), root, maxProgression);
+
+        registerEvent(PlayerStatisticIncrementEvent.class, e -> {
+            if (e.getStatistic() == Statistic.WALK_ONE_CM) {
+                if (e.getNewValue() >= 160934.4 * 1000) {
```

Comment:
i would seperate this multiplication a bit more
100*1000*1609.344, or have it in meters
100*1609344